src/Hl7.Fhir.Base/ElementModel/TypedElementParseExtensions.cs

```diff
@@ -33,27 +35,34 @@ public static class TypedElementParseExtensions
         ///   'string' => code
         ///   'uri' => code
         /// </remarks>
-        public static Element ParseBindable(this ITypedElement instance)
+        public static Element? ParseBindable(this ITypedElement instance)
 #pragma warning disable CS0618 // Type or member is obsolete
             => instance.ParseBindableInternal();
 #pragma warning restore CS0618 // Type or member is obsolete
 
         [Obsolete("WARNING! Intended for internal API usage exclusively, interface IBaseElementNavigator can be changed in " +
             "the near future.")]
-        public static Element ParseBindableInternal<T>(this IBaseElementNavigator<T> instance) where T : IBaseElementNavigator<T>
+        internal static Element? ParseBindableInternal<T>(this IBaseElementNavigator<T> instance) where T : IBaseElementNavigator<T>
```

Comment:
Yep, the method is even called "....Internal" and should absolutely not be public, since it will never become public, even when we open up the lower level API!